service/src/repository/user_respository.ts

```diff
@@ -0,0 +1,255 @@
+import {
+    CreateTableCommand,
+    DeleteItemCommand, DeleteTableCommand,
+    PutItemCommand,
+    ScanCommand,
+    UpdateItemCommand
+} from "@aws-sdk/client-dynamodb";
+import {User} from "../graphql/schema";
+
+import {DynamoDBClient, GetItemCommand} from "@aws-sdk/client-dynamodb";
+import { uuid } from 'uuidv4';
+
+// Create DynamoDB service object
+const defaultConfig : {
+    region: string,
+    endpoint?: string,
+} = {
+    region: "us-west-2"
+};
+
+if (process.env.TEST_MODE) {
```

Comment:
Can you think of another design to avoid having test code inside your application repositories?